packages/commercetools/composables/src/useUser/index.ts

```diff
@@ -61,11 +65,24 @@ export default function useUser(): UseUser<Customer> {
     cart.value = null;
   };
 
+  const changePassword = async (currentPassword: string, newPassword: string) => {
+    loading.value = true;
+    try {
+      const userResponse = await customerChangeMyPassword(user.value.version, currentPassword, newPassword);
```

Comment:
You can use here Promise.all to both queries

Reply:
Actually I can not, those have to run in sequence.